SDWebImage/SDWebImageWebPCoder.m

```diff
@@ -259,7 +230,7 @@ - (UIImage *)decompressedImageWithImage:(UIImage *)image
     return image;
 }
 
-- (nullable UIImage *)sd_blendWebpImageWithCanvas:(CGContextRef)canvas iterator:(WebPIterator)iter {
+- (nullable UIImage *)sd_drawnWebpImageWithCanvas:(CGContextRef)canvas iterator:(WebPIterator)iter {
```

Comment:
These two method is nearly the same except that blend mode process. For non-blend(Which means you should **cover** the current frame imaeg), just clear the target image rectangle before draw.